tests/permissions/permissions.csv

```diff
@@ -54,11 +54,17 @@ true,,,q,(`a`b`c!1 4 7)~.pm.expr[`tom;"first each d"],,,test expression with eac
 fail,,,q,.pm.expr[`harry;"first each d"],,,harry doesn't have access to d
 
 comment,,,,,,,test passing a symbol argument to both a lambda function and a projection
-true,,,q,.pm.expr[`tom;({[x;y]y~`ABCD};`x;`ABCD)],,,passing a symbol argument to a lambda function
-true,,,q,.pm.expr[`tom;({[x;y]y~`ABCD}[`x];`ABCD)],,,passing a symbol argument to a projection
+true,,,q,.pm.requ[`tom;({[x;y]y~`ABCD};`x;`ABCD)],,,passing a symbol argument to a lambda function
+true,,,q,.pm.requ[`tom;({[x;y]y~`ABCD}[`x];`ABCD)],,,passing a symbol argument to a projection
 
 comment,,,,,,,test lambdas and string-wrapped lambdas and projections are treated the same
 true,,,q,.pm.requ[`tom; ({x+a};1)]~.pm.requ[`tom; ("{x+a}";1)],,,passing lambda as string returns same as regular lambda
 true,,,q,.pm.requ[`tom; ({x+y}[1];2)]~.pm.requ[`tom; ("{x+y}[1]";2)],,,passing projection as string returns same as regular projection
-fail,,,q,.pm.requ[`harry; ({x+a};1)]~2,,,harry cannot run regular lambda which accesses var a
-fail,,,q,.pm.requ[`harry; ("{x+a}";1)]~2,,,harry cannot run string lambda which accesses var a
\ No newline at end of file
+fail,,,q,.pm.expr[`harry; ({x+a};1)]~2,,,harry cannot run regular lambda which accesses var a
+fail,,,q,.pm.expr[`harry; ("{x+a}";1)]~2,,,harry cannot run string lambda which accesses var a
+
+comment,,,,,,,test defined projection
+run,,,q,fn:{x+y}[1],,,creating a projection 
+true,,,q,".pm.pdict[`fn;1]~(enlist[`]!enlist[::]),(`x`y)!1 1",,,check output is as expected
```

Comment:
I don't think we need this test - this is an implementation detail rather than an important behaviour. as long as `.pm.expr` works, for the tests we don't need to worry about _how_ it works